pkg/kubelet/cm/topologymanager/scope_container.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package topologymanager
 
 import (
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Can we drop this?

Reply:
yep, same as https://github.com/kubernetes/kubernetes/pull/101432#discussion_r624933346